src/components/ButtonPanel.js

```diff
@@ -1,42 +1,47 @@
+import PropTypes from 'prop-types';
 /* eslint-disable no-unused-vars */
```

Comment:
## [OPTIONAL] Stretch Requirement

- [ ] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).